### PR TITLE
Update modules to fix security vulns

### PR DESCRIPTION
**Pull request by psoleckimoj**
- Jade has become pug
- Winston has had breaking changes

### CODE REVIEW

**Review by davidthomas22 — DISMISSED** [review]
LGTM

File: log.js [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
 } else {
-    logger.remove(logger.transports.Console);
-    logger.add(logger.transports.Console, {
-        prettyPrint: true,
-        colorize: true,
-        silent: false,
-        timestamp: true
-        // json: true
-    });
+    logger.add(new transports.Console());
```
Comment (mattops) [inline log.js:22]: you sure you want to log to the console?  Must have been a reason it was removed.

Comment (psoleckimoj) [reply log.js:22]: I'm not sure how much these are used as it's just a local mock sso instance as far as I can see?

Comment (mattops) [reply log.js:22]: Look fine either way.  Just realised this is the mock sso app so it's not deployed anywhere.

**Review by mattops — DISMISSED** [review]

**Review by mattops — APPROVED** [review]

**Pull request merged**